chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -397,199 +399,207 @@
+      else
+        node.set[:provisioner][:repositories][os][arch]["base"] = { "baseurl=http://#{admin_ip}:#{web_port}/#{os}/install/Server" => true }
+      end
+      # Default kickstarts and crowbar_join scripts for redhat.
+
+      template "#{os_dir}/crowbar_join.sh" do
+        mode 0644
+        owner "root"
+        group "root"
+        source "crowbar_join.redhat.sh.erb"
+        variables(admin_web: admin_web,
+                  os_codename: os_codename,
+                  crowbar_repo_web: crowbar_repo_web,
+                  admin_ip: admin_ip,
+                  provisioner_web: provisioner_web,
+                  web_path: web_path)
+      end
 
-    template "#{os_dir}/net-post-install.sh" do
-      mode 0644
-      owner "root"
-      group "root"
-      variables(admin_web: admin_web,
-                os_codename: os_codename,
-                repos: node[:provisioner][:repositories][os],
-                admin_ip: admin_ip,
-                provisioner_web: provisioner_web,
-                web_path: web_path)
-    end
+    when /^ubuntu/ =~ os
+      node.set[:provisioner][:repositories][os][arch]["base"] = { "http://#{admin_ip}:#{web_port}/#{os}/install" => true }
+      # Default files needed for Ubuntu.
+
+      template "#{os_dir}/net-post-install.sh" do
+        mode 0644
+        owner "root"
+        group "root"
+        variables(admin_web: admin_web,
+                  os_codename: os_codename,
+                  repos: node[:provisioner][:repositories][os][arch],
+                  admin_ip: admin_ip,
+                  provisioner_web: provisioner_web,
+                  web_path: web_path)
+      end
 
-    template "#{os_dir}/crowbar_join.sh" do
-      mode 0644
-      owner "root"
-      group "root"
-      source "crowbar_join.ubuntu.sh.erb"
-      variables(admin_web: admin_web,
-                os_codename: os_codename,
-                crowbar_repo_web: crowbar_repo_web,
-                admin_ip: admin_ip,
-                provisioner_web: provisioner_web,
-                web_path: web_path)
-    end
+      template "#{os_dir}/crowbar_join.sh" do
+        mode 0644
+        owner "root"
+        group "root"
+        source "crowbar_join.ubuntu.sh.erb"
+        variables(admin_web: admin_web,
+                  os_codename: os_codename,
+                  crowbar_repo_web: crowbar_repo_web,
+                  admin_ip: admin_ip,
+                  provisioner_web: provisioner_web,
+                  web_path: web_path)
+      end
 
-  when /^(hyperv|windows)/ =~ os
-    template "#{tftproot}/adk-tools/build_winpe_#{os}.ps1" do
-      mode 0644
-      owner "root"
-      group "root"
-      source "build_winpe_os.ps1.erb"
-      variables(os: os,
-                admin_ip: admin_ip)
-    end
+    when /^(hyperv|windows)/ =~ os
+      # Windows is x86_64-only
+      os_dir="#{tftproot}/#{os}"
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `=`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)